app/main.py

```diff
@@ -30,3 +50,60 @@ def profile(request: Request):
         "username": current_username,
         "events": upcouming_events
     })
+
+
+@app.get("/profile/{user_id}/EditEvent", response_class=HTMLResponse)
```

Comment:
Prefer adding this endpoint on the relevant file on app/routers. The `main.py` file should be clean.